package-lock.json

```diff
@@ -413,10 +413,13 @@
       }
     },
     "node_modules/@brightspace-ui/intl": {
-      "version": "3.16.1",
-      "resolved": "https://registry.npmjs.org/@brightspace-ui/intl/-/intl-3.16.1.tgz",
-      "integrity": "sha512-KsoUKvy4LFkW9XQeTbTf5G9DyMZeFwDIDXh1LmaKLYBCJpVbUznzG+ZLxv3xrmB31lQlZF4fnP6ouN1uoJIz5w==",
-      "license": "Apache-2.0"
+      "version": "3.17.0",
+      "resolved": "https://registry.npmjs.org/@brightspace-ui/intl/-/intl-3.17.0.tgz",
+      "integrity": "sha512-fBpSp9EEWrHYRgIoz/X4eiW6statL4WmqTdkesTj9OIC6abBq0g87UwVLPom4oYFWudEgtkRvJre0DOhz0m/xA==",
+      "dependencies": {
+        "@formatjs/intl-pluralrules": "^1",
```

Comment:
Side note this version is super duper old, but hasn't been getting updates for whatever reason. Maybe we ignored them a long time ago. When I tried it caused errors, so I'll look into updating this along with validating Maori support.